eQuiz/modules/eQuiz.Web/Views/Home/MainLogin.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
In my opinion current view just duplicates the https://github.com/RDLviv/eQuiz/blob/master/eQuiz/modules/eQuiz.Web/Views/Account/Index.cshtml.

Is it correct?